Ms2Database/DBClasses/Ms2DBContext.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Data.Entity;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Ms2Database.DBClasses
+{
+    public class Ms2DBContext : DbContext
```

Comment:
Can we have consistent naming here? `DB` in one case, then `Db` in another case

Reply:
will be addressed

Reply:
has been changed.